src/main/resources/dbchangelogs/changelogmaster.xml

```diff
@@ -81,4 +81,5 @@
   <include file="dbchangelogs/changelog-4.15.0-add-upcoming-vacation-reminder.xml"/>
   <include file="dbchangelogs/changelog-4.15.0-add-to-allow-half-day-absences.xml"/>
   <include file="dbchangelogs/changelog-4.16.0-add-holiday-replacement.xml"/>
+  <include file="dbchangelogs/changelog-4.22.0-add-end-of-sickpay-notification-send.xml"/>
```

Comment:
ich dachte wir packen da noch in die 4.20.6 rein, da es ja fertig ist und warum auch nicht :)